lxd/instance/drivers/driver_lxc.go

```diff
@@ -7958,66 +7942,6 @@ func (d *lxc) removeDiskDevices() error {
 	return nil
 }
 
-// Network I/O limits.
-func (d *lxc) setNetworkPriority() error {
-	// Load the go-lxc struct.
-	cc, err := d.initLXC(false)
-	if err != nil {
-		return err
-	}
-
-	// Load the cgroup struct.
-	cg, err := d.cgroup(cc, true)
-	if err != nil {
-		return err
-	}
-
-	// Check that the container is running
-	if d.InitPID() <= 0 {
-		return fmt.Errorf("Can't set network priority on stopped container")
-	}
-
-	// Don't bother if the cgroup controller doesn't exist
-	if !d.state.OS.CGInfo.Supports(cgroup.NetPrio, cg) {
-		return nil
-	}
-
-	// Extract the current priority
-	networkPriority := d.expandedConfig["limits.network.priority"]
-	if networkPriority == "" {
-		networkPriority = "0"
-	}
-
-	networkInt, err := strconv.Atoi(networkPriority)
-	if err != nil {
-		return err
-	}
-
-	// Get all the interfaces
-	netifs, err := net.Interfaces()
-	if err != nil {
-		return err
-	}
-
-	// Check that we at least succeeded to set an entry
-	success := false
-	var lastError error
-	for _, netif := range netifs {
-		err = cg.SetNetIfPrio(fmt.Sprintf("%s %d", netif.Name, networkInt))
```

Comment:
Do we also need to remove this function?

Reply:
Do you mean `cg.SetNetIfPrio`? I've removed it because it was no longer in use

Reply:
So you have, i missed it, sorry.